### ISSUE TITLE
Want feature to modify markdown file based on memorization data

### ISSUE THREAD

**Issue by Brandon-Lu737** [ankitects/anki#2890]
**- Have a question or feature suggestion?**

Hi, I'm a developer and I'm trying to build a plug-in that syncs Anki's data back to LogSeq. Basically, we already have a plug-in in LogSeq that connects to Anki via AnkiConncet to export all the flashcards in LogSeq to Anki. So we complete the step of exporting "LogSeq-> Anki".

But the other way around is incomplete. If we study a card and update its time to show up the next time, I was wondering if there's a way of updating that back to the LogSeq files. In logSeq for example, you have some `metadata` for each `card block` in the markdown file that shows you how well you know the card.

Basically, I'm asking to import the progress of cards from Anki to LogSeq so that not every card is a brand new card.

**I was wondering if anyone can point to a place where I can get started on this feature. Thanks.**

**Comment by dae** [comment]
You should be able to accomplish that in an add-on. You can find a brief intro to writing them here: https://addon-docs.ankiweb.net/intro.html
